.github/workflows/e2e-commit-tests.yml

```
@@ -0,0 +1,154 @@
+          export CY_SKIP_TAGS="@Bug,@Maintain,@NonConcurrent"
+
+          echo "🎯 Running tests with tags: $CY_TEST_TAGS"
+          echo "🚫 Skipping tags: $CY_SKIP_TAGS"
+
+          npm run test:cypress:e2e:ci
+
+      - name: Upload test results
+        if: always()
+        uses: actions/upload-artifact@v4
+        with:
+          name: e2e-test-results-${{ needs.pr-comment-parser.outputs.test-type }}-${{ matrix.test-set }}
```

actionlint (1.7.7)

property "pr-comment-parser" is not defined in object type {}

(expression)